### PR TITLE
correct the description of 'no_of_packets' in TARDIS Documentation

### PR DESCRIPTION
**Pull request by grgkaran03**
### :pencil: Description

**Type:** : 📝

Description of no_of_particles parameter was incorrect in TARDIS documentation[ https://tinyurl.com/ef64s538 ]. So I looked up and corrected the description. 

closes https://github.com/tardis-sn/tardis/issues/2066#issue-1280388091

### :pushpin: Resources

https://tinyurl.com/ef64s538

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [X] My changes can't be tested ( Just corrected the text in .yml file. Don't think it needs tests.)

### :ballot_box_with_check: Checklist

- [x] I requested two reviewers for this pull request
- [X] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: tardis/io/schemas/montecarlo.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ properties:
   no_of_packets:
     type: number
     multipleOf: 1.0
-    description: Seed for the random number generator
+    description: This gives the number of packets normally used in each iteration.
```
Comment (smithis7) [inline tardis/io/schemas/montecarlo.yml:20]: I don't fully agree with the word "normally" here. I would specify that it is used for all but the final iteration.

Comment (grgkaran03) [reply tardis/io/schemas/montecarlo.yml]: Thanks for reviewing!
Will look into it for corrections...

**Comment by grgkaran03** [comment]
@smithis7 
please review the corrections...

**Comment by smithis7** [comment]
@andrewfullard is this a good description? If you think so I'll approve.

**Pull request closed without merging**